tests/integration/test_properties.py

```diff
@@ -215,15 +216,20 @@ def test_delete_property_by_id(client, auth_headers, test_database):
     assert response == 404
 
 
-def test_update_property_by_id(client, auth_headers, create_property, test_database):
+def test_update_property_by_id(client, empty_test_db, valid_header, create_property):
```

Comment:
I think this method can be renamed to `test_update`. The `by id` part is a bit misleading and confusing.

Reply:
Thanks for updating to use the new faster factories! 🎉